hathor/conf/settings.py

```diff
@@ -247,3 +247,6 @@ def MAXIMUM_NUMBER_OF_HALVINGS(self) -> int:
 
     # Amount in which tx min weight reaches the middle point between the minimum and maximum weight
     MIN_TX_WEIGHT_K: int = 100
+
+    # Quantity of blocks without a weight verification when doing a full verification
+    QUANTITY_BLOCKS_SKIP_WEIGHT_VERIFICATION: int = 1000
```

Comment:
This name is a bit long, that would be OK if the meaning was more clear. The comment can be interpreted as "don't verify 1000 blocks, then verify the rest", but I understood it as "don't verify any blocks, unless the last 1000". Turns out was neither.

I think including "every n" in some form would be enough (at least for my understanding). Some suggestions:

- `SKIP_VERIFY_WEIGHT_EXCEPT_EVERY_N_BLOCKS`: maybe a bit long but in my head this is literally what it does;
- `VERIFY_WEIGHT_EVERY_N_BLOCKS`: avoids double negation (skip+except);
- `BLOCK_VERIFY_WEIGHT_EVERY_N`: ordered more "hierarchical" (`block.verify_weight(every_n)`) but not grammatical;

What do you think?

Reply:
I agree with you that the name is not good. I do like the second option (`VERIFY_WEIGHT_EVERY_N_BLOCKS`). Changed for that one.